setup.py

```diff
@@ -77,6 +77,7 @@
 install_requires = [
     'Flask-BabelEx>=0.9.2',
     'Flask>=0.11.1',
+    'celery<4.0,>=3.1',
```

Comment:
it should not be necessary if yes then https://github.com/inveniosoftware/invenio-webhooks/pull/48/files#diff-2eeaed663bd0d25b7e608891384b7298R49 should be removed.